src/javascript/app/pages/trade/markets.jsx

```diff
@@ -68,11 +68,19 @@ const List = ({
 class Markets extends React.Component {
     constructor (props) {
         super(props);
-        let market_symbol = Defaults.get('market');
+        let market_symbol,underlying_symbol;
```

Comment:
please check the syntax of how we declare variables. there should be a new line after the comma.

Reply:
2 variables declared in two lines.